packages/client/tests/functional/methods/findUniqueOrThrow/tests.ts

```diff
@@ -27,31 +27,37 @@ testMatrix.setupTestSuite((_suiteConfig, _suiteMeta, clientMeta) => {
     await expect(record).rejects.toMatchObject(new Prisma.NotFoundError('No User found', '0.0.0'))
   })
 
-  testIf(clientMeta.runtime !== 'edge')('works with transactions', async () => {
-    const newEmail = faker.internet.email()
-    const result = prisma.$transaction([
-      prisma.user.create({ data: { email: newEmail } }),
-      prisma.user.findUniqueOrThrow({ where: { email: nonExistingEmail } }),
-    ])
+  skipTestIf(clientMeta.runtime === 'edge' || _suiteConfig.driverAdapter === 'js_d1')(
```

Comment:
Isn't `_suiteConfig.driverAdapter === 'js_d1'` is already covered by `clientMeta.runtime === 'edge'`?

Reply:
It's needed actually
Example
```
pnpm run test:functional:code --preview-features=driverAdapters --adapter=js_d1 --runInBand findUniqueOrThrow
```
Without the condition, would fail (which is expected and why it needs to be skipped)